libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js

```diff
@@ -56,21 +45,20 @@ describe('Schema Merge Tests', function () {
     // This fixture creates or updates test.schema by calling bf dialog:merge on all the schema files in the solution.
     // This will install the latest version of botframework-cli if the schema changed and npm is present.
     // This only runs on Windows platforms.
-    maybeIt('should merge dialog schemas', async function () {
+    it('should merge dialog schemas', async function () {
         this.timeout(150000);
         fs.unlinkSync(testsSchemaPath);
 
         // Merge all schema files.
-        const sdkRoot = path.join(__dirname, '..', '..', '..');
         const mergeCommand = [
-            `bf dialog:merge "${sdkRoot}/libraries/**/*.schema"`,
-            `"${sdkRoot}/libraries/**/*.uischema"`,
-            '!**/testbot.schema',
-            '!**/botbuilder-dialogs-adaptive/tests/schema/*.*',
+            'bf dialog:merge ./libraries/**/*.schema',
+            './libraries/**/*.uischema',
+            '"!"**/testbot.schema',
+            '"!"**/botbuilder-dialogs-adaptive/tests/schema/*.*',
```

Comment:
Note: `"!"` was required to get this to work in Bash. Thankfully, it still works just fine in CMD and PWSH.